### PR TITLE
Clayton/migrate/vite react

### PR DESCRIPTION
**Pull request by Clayton0024**
Migration to vite & react for front end.

### CODE REVIEW

File: Frontend/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+npm  install
+
+# Basic Commands
+
+npm run dev (start dev server)
+
+npm run build (build for production)
+
+npm run serve (serving the production build locally)
```
Comment (brha8286) [inline Frontend/README.md:23]: + `npm run lint`

File: Frontend/src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+function App() {
+  const [count, setCount] = useState(0)
+
+  return (
+    <>
+      <div>
+        <a href="https://vitejs.dev" target="_blank">
+          <img src={viteLogo} className="logo" alt="Vite logo" />
+        </a>
+        <a href="https://react.dev" target="_blank">
```
Comment (brha8286) [inline Frontend/src/App.jsx:15]: Fails lint

File: Frontend/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+
+
+# Navigate to the project directory
+
+cd Frontend
+
+# Install dependencies
+
+npm  install
```
Comment (brha8286) [inline Frontend/README.md:15]: Include required versions of node, npm, react, etc.

**Review by brha8286 — CHANGES_REQUESTED** [review]

File: Frontend/README.md (hunk 1 above)
Comment (Clayton0024) [reply Frontend/README.md]: updated

File: Frontend/src/App.jsx (hunk 2 above)
Comment (Clayton0024) [reply Frontend/src/App.jsx]: removed that return and made return basic element as it gets overwritten later.

File: Frontend/src/App.jsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import "./App.css";
+
+function App() {
+  return (
+    <>
+      <p>Hello World</p>
```
Comment (brha8286) [inline Frontend/src/App.jsx:6]: perfect

**Review by brha8286 — APPROVED** [review]

**Pull request merged**